src/mantine-hooks/src/use-mouse/use-mouse.ts

```diff
@@ -1,7 +1,7 @@
 import { useEffect, useRef, useState } from 'react';
 import type { MouseEvent } from 'react';
 
-export function useMouse<T extends HTMLElement = any>() {
+export function useMouse<T extends HTMLElement = any>(resetOnExit: boolean = false) {
```

Comment:
options should be an object: `options: { resetOnExit?: boolean } = { resetOnExit: false }`

Reply:
Done